requirements/dev.txt

```diff
@@ -10,10 +10,12 @@ attrs==19.3.0             # via black, pytest
 babel==2.7.0              # via sphinx
 backcall==0.1.0           # via ipython
 black==19.10b0
-chardet==3.0.4            # via doc8
+certifi==2019.11.28       # via requests
```

Comment:
What prompted these changes to requirements?

Reply:
possibly we updated cumulusci in the past without re-running pip-compile to update the pins for its dependencies, or something like that.

Reply:
or maybe we only ran it for prod.txt and forgot dev.txt (which extends prod)

Reply:
Ok -- I re-locked in `gh-commits` branch.